standalone/src/models/product-selection/product-of-selection/index.ts

```diff
@@ -0,0 +1,9 @@
+import { GraphqlModelBuilder } from './builders';
+import * as ProductOfSelectionPresets from './presets';
+export * from '../types';
```

Comment:
We must export the types of this model:
```suggestion
export * from './types';
```

Reply:
Just to clarify, in this case we don't need to export types from the main index file (product-selection/index), only from this file, for the same reason we have `types` file separately for this submodel, right @CarlosCortizasCT ?